framework/mesh/mesh_generator/mesh_generator.cc

```diff
@@ -177,31 +198,13 @@ MeshGenerator::PartitionMesh(const UnpartitionedMesh& input_umesh, int num_parti
     }
   }
 
-  // Note A: We do not add the diagonal here. If we do it, ParMETIS seems
+  // Note A: We do not add the diagonal here. If we do, ParMETIS seems
```

Comment:
I know it is not your comment, but what does this mean? And what about PTSCOTCH then? Should we create an issue to remember to investigate the meaning of that comment? It seems important for parallel efficiency.

Reply:
I'm not sure what this comment means. I'm not even sure it's still relevant to the code that it references. It doesn't look to me like we are excluding anything. I'll open an issue to investigate what's going on here.

Reply:
see email